src/formElementPlugins/useDefault.ts

```diff
@@ -30,11 +30,22 @@ const useDefault = ({
     // This prevents an existing response from being wiped out by a default on
     // first load, but allows it to still be replaced on subsequent default
     // changes
-    setIsInitialLoad(false)
+    if (isInitialLoad) {
+      if (!currentResponse?.text) {
+        onChange(defaultValue)
+      }
+      setIsInitialLoad(false)
+      return
+    }
 
-    if ((!persistUserInput || !currentResponse?.text) && !isInitialLoad) {
+    if (!currentResponse?.text) {
       onChange(defaultValue)
+      return
     }
+
+    if (persistUserInput) return
+
+    onChange(defaultValue)
```

Comment:
This should fix the default behaviour, both for existing defaults, and inside listBuilder.